.changeset/fluffy-beans-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": major
```

Comment:
Should not be a major but rather a minor

```suggestion
"ledger-live-desktop": minor
```

Reply:
Thanks!